src/api/Jalasoft.TeamUp.Projects.API/Controllers/PostProject.cs

```diff
@@ -39,21 +39,19 @@ public PostProject(IProjectsService postProjectService)
                 var result = this.postProjectService.PostProject(data);
                 return new CreatedResult("v1/projects/:id", result);
             }
+            catch (ValidationException exVal)
+            {
+                var errorException = new ProjectsException(ProjectsErrors.BadRequest, exVal);
+                return errorException.Error;
+            }
             catch (ProjectsException e)
             {
-                return new ContentResult
-                {
-                    StatusCode = e.StatusCode,
-                    Content = e.ProjectsErrorMessage,
-                };
+                return e.Error;
             }
-            catch (System.Exception)
+            catch (System.Exception e)
```

Comment:
minor: use the 'using System' so this can be used as Exception only

Reply:
done